src/client/game.cpp

```diff
@@ -2370,6 +2371,18 @@ void Game::updateCameraDirection(CameraOrientation *cam, float dtime)
 	}
 }
 
+// Get the factor to multiply with sensitivity to get the same mouse/joystick
+// responsiveness independently of FOV.
+f32 Game::getSensitivityScaleFactor() const
+{
+	f32 fov_y = client->getCamera()->getFovY();
+
+	// Multiply by a constant such that it becomes 1.0 at 72 degree FOV and
+	// 16:9 aspect ratio to minimize disruption of existing sensitivity
+	// settings.
+	return tan(fov_y / 2.0f) * 1.3763818698f;
```

Comment:
this constant should be a DEFINE in order to understand what is this directly

Reply:
I found it hard to give such a constant a meaningful name so I thought it was more clear just to multiply with it and explain its purpose in the comments. If `std::tan` was `constexpr` in C++11 I could do the something like following which would be more clear about its purpose and where it comes from:

```
static constexpr float default_fov_y = 72.0f * M_PI / 180.0f;
static constexpr float constant = 1.0f / tan(default_fov_y / 2.0f);
```

This works for gcc but as far as I know `std::tan` isn't is not guaranteed to be `constexpr` in C++11.

Maybe you have a better idea what to call such a constant?

Also where should I put it? I think it is better to have a `constexpr` variable in the function instead of a global macro because it is only used in that function.

Reply:
Personally, I think the comment right above it good enough to explain it.